openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -0,0 +1,52 @@
+"""
+This file contains utility functions which will responsible for sending emails.
+"""
+import logging
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.mail import send_mail
+from django.utils.translation import ugettext as _
+
+from edxmako.shortcuts import render_to_string
+from microsite_configuration import microsite
+from openedx.core.djangoapps.user_api.accounts.api import get_account_settings
+from xmodule.modulestore.django import modulestore
+
+
+log = logging.getLogger(__name__)
+
+
+def send_credit_notifications(username, course_key, email_type="credit_eligibility"):
+    """Sends email notification to user on different phases during credit
+    course e.g. credit eligibility, credit payment etc.
+    """
+    user = None
+    try:
+        user = User.objects.get(username=username)
+    except User.DoesNotExist:
+        log.error('No user with %s exist', username)
+        return
+
+    account_settings = get_account_settings(user)
+    course = modulestore().get_course(course_key, depth=0)
+    course_display_name = course.display_name
```

Comment:
Based on a look at the documentation, the data model does seem to support it; however I'm not sure if/how that API is meant to be used in-process.  @clintonb ?